.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+registries:
+  public-nuget:
+    type: nuget-feed
+    url: https://api.nuget.org/v3/index.json
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 5
+    labels:
+      - area-infrastructure
```

Comment:
I wonder if we should use a different area label for repository automation, separating that from "infrastructure", perhaps creating `area-github-automation`?

Reply:
Yeah, `area-codeflow` is OK with me.